core/spec.md

```diff
@@ -6453,6 +6558,15 @@ as appropriate, including being specified in a language other than English.
 
 <!-- start-err-def -->
 
+#### ancestor_circular_reference
+
+* Type: `https://github.com/xregistry/spec/blob/main/core/spec.md#ancestor_circular_reference`
+* Code: `400 Bad Request`
+* Instance: The request URL
```

Comment:
Let's make this a URL to the Version we're touching

Reply:
in a large/tree update, it might hard to know which version we're talking about if more than one is trying to use the same ancestor value